nursery_store.go

```diff
@@ -478,13 +436,6 @@ func (ns *nurseryStore) PreschoolToKinder(kid *kidOutput) error {
 			maturityHeight = kid.ConfHeight() + kid.BlocksToMaturity()
 		}
 
-		// In the case of a Late Registration, we've already graduated
-		// the class that this kid is destined for. So we'll bump its
-		// height by one to ensure we don't forget to graduate it.
-		lastGradHeight, err := ns.getLastGraduatedHeight(tx)
-		if err != nil {
-			return err
-		}
```

Comment:
Appears that the fragment below won't compile now due to this change.

Reply:
`lastGradHeight` is now a function arg.